test-integration/filter-aave-v2.js

```diff
@@ -0,0 +1,83 @@
+/* global artifacts */
+
+const { assert } = require("chai");
+const { deployArgent } = require("../utils/argent-deployer.js");
+const utils = require("../utils/utilities.js");
+
+const AaveV2Filter = artifacts.require("AaveV2Filter");
+const AaveV2LendingPool = artifacts.require("AaveV2LendingPoolMock");
+
+const amount = web3.utils.toWei("1");
+
+contract("Aave V2 Filter", (accounts) => {
+  let argent;
+  let wallet;
+  let pool;
+
+  before(async () => {
+    argent = await deployArgent(accounts);
+
+    pool = await AaveV2LendingPool.at("0x7d2768dE32b0b80b7a3454c06BdAc94A69DDc7A9");
+    const filter = await AaveV2Filter.new();
+    await argent.dappRegistry.addDapp(0, pool.address, filter.address);
+  });
+
+  beforeEach(async () => {
+    wallet = await argent.createFundedWallet({ DAI: amount });
+  });
+
+  const deposit = async (beneficiary) => argent.multiCall(wallet, [
```

Comment:
We could check that the balances have been updated as part of the `deposit` and `withdraw` methods.